packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/DateTimePickerOnSelect.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import DateTimePicker from '../../../DateTimePicker';
+
+class DatePickerOnChange extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { onchangedate: '', onselectdate: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+    this.handleDateSelect = this.handleDateSelect.bind(this);
+  }
+
+  handleDateChange(event, onchangedate) {
+    this.setState({ onchangedate });
+  }
+
+  handleDateSelect(event, onselectdate) {
+    this.setState({ onselectdate });
+  }
+
+  render() {
+    return (
+      <div>
+        <h3>
+OnChange Date:
+          <span id="selected-date">{this.state.onchangedate}</span>
```

Comment:
nit: Perhaps, choose an id that is different from the one below. (i.e. "changed-date"). But it doesn't affect anything here.

Reply:
Fixed in https://github.com/cerner/terra-framework/pull/585/commits/5e89405a15d090636a60395b922c116a068c2f58